src/components/NodeLink/functionUpdateVis.ts

```diff
@@ -205,7 +205,7 @@ export function showTooltip(message: string, event: MouseEvent, delay = 200) {
   tooltip.html(message)
     .style('left', `${event.clientX + 10}px`)
     .style('top', `${event.clientY - 20}px`);
-  tooltip.transition().duration(delay).style('opacity', 0.9);
+  tooltip.transition().duration(delay).style('opacity', 1);
```

Comment:
Why change this? Seems like there's some value in remaining aware of the matrix view underneath the tooltip? It's ok to change it to opacity 1, but I was curious about your rationale.

Reply:
It just looked better without transparency. More readable